### ISSUE TITLE
evaluate result

### ISSUE THREAD

**Issue by huidemie** [facebookresearch/wsd-biencoders#4]
Hello, I run your code directly, but got evaluate F1  is 77.5 on All dataset. Could you tell me why ?

**Comment by blvns** [comment]
Hi! Can you tell me a little more about the experiment you ran? Did you retrain the model using the training script and default parameters?

**Comment by huidemie** [comment]
> Hi! Can you tell me a little more about the experiment you ran? Did you retrain the model using the training script and default parameters?

I ran training script by this command : "python biencoder.py --data-path ../data/All_Words_WSD --ckpt ./wsd-biencoder". I put the chekpoint file you shared into the ckpt folder. And I used default parameters.

**Comment by blvns** [comment]
Are you trying to evaluate the provided model checkpoint or retrain the model from scratch? 

If evaluating the checkpoint, the full evaluation command is:
`python biencoder.py --data-path $path_to_wsd_data --ckpt $path_to_model_checkpoint --eval --split $wsd_eval_set`
You would need to include the --eval flag, otherwise the script will train a new model on the WSD dataset.

If you are retraining the model, the script won't use the provided checkpoint and will retrain the model from scratch. This may lead to a different final F1 on the test set than what we reported -- unfortunately, PyTorch isn't reproducible across versions or on different hardware when there is randomness (https://pytorch.org/docs/stable/notes/randomness.html, https://github.com/pytorch/pytorch/issues/7068), even if the random seeds are set.

**Comment by huidemie** [comment]
OK, I know. Thanks a lot !
